File: Desenvolvimento/server/src/ApplicationCore/Entities/FluxoItemChecklist.cs
using ApplicationCore.Interfaces.Base;
using System;
using System.Collections.Generic;
using System.Text;

namespace ApplicationCore.Entities
{
    public class FluxoItemChecklist : IBaseEntity<long>
    {
        public long Id { get; set; }
        public string Nome { get; set; }

        public long FluxoItemId { get; set; }
        public virtual FluxoItem FluxoItem { get; set; }

        public virtual ICollection<TramiteChecklist> TramiteChecklists { get; set; }
    }
}

File: Desenvolvimento/client/src/@fuse/types/models/fluxo-item-checklist.ts
import { FluxoItem } from './fluxo-item';
import { BaseModel } from '../base-model';
import { TramiteChecklist } from './tramite-check-list';

export class FluxoItemCheckList  extends BaseModel {
    constructor(
        public id: number,
        public nome?: string,
        public fluxoItemId?: number,
        public fluxoItem?: FluxoItem,
        public tramiteChecklists?: Array<TramiteChecklist>
    ) { super(); }
}
